ansible/configs/ocp-ha-lab/env_vars.yml

```diff
@@ -169,7 +175,7 @@ ansible_ssh_private_key_file: ~/.ssh/{{key_name}}.pem
 
 project_tag: "{{ env_type }}-{{ guid }}"
 
-docker_version: "1.12.6"
+docker_version: "1.13.1"
```

Comment:
don't you want to use your magic line here?

    docker_version: "{{ '1.12.6' if repo_version | version_compare('3.9', '<')  else '1.13.1' }}"

Reply:
We don't really need to keep ocp-ha-lab backward compatible.